blueoil/converter/templates/Makefile

```diff
@@ -5,7 +5,7 @@ INPUTS_SRC_DIR := ./src/inputs
 .PHONY: help
 help:
 	@echo make clean
-	@echo make build ARCH={x86, x86_avx, aarch64, arm, fpga} TYPE={executable, dynamic, static}
+	@echo make build ARCH={x86, x86_avx, aarch64, arm, fpga, fpga_aarch64} TYPE={executable, dynamic, static}
```

Comment:
How about to introduce a flag like `USE_FPGA` ?

Reply:
@primenumber 
USE_FPGA looks good because it's easy to remember.
I'll try to use it instead.